presto-main/src/test/java/com/facebook/presto/sql/planner/TestEffectivePredicateExtractor.java

```diff
@@ -440,18 +440,25 @@ public void testInnerJoin()
                 null
         );
 
+        FilterNode left = filter(leftScan,
+                and(
+                        lessThan(BE, AE),
+                        lessThan(CE, bigintLiteral(10)),
+                        equals(GE, bigintLiteral(10))));
+        FilterNode right = filter(rightScan,
+                and(
+                        equals(DE, EE),
+                        lessThan(FE, bigintLiteral(100))));
+
         PlanNode node = new JoinNode(newId(),
                 JoinNode.Type.INNER,
-                filter(leftScan,
```

Comment:
It's not a format change. I had to extract the nested nodes so that I can reference them below to get their output symbols.